Gems/WhiteBox/Code/Source/Viewport/WhiteBoxPolygonTranslationModifier.h

```diff
@@ -24,6 +24,9 @@ namespace WhiteBox
 {
     class ManipulatorViewPolygon;
 
+    // small offset to the polygon manipulator to prevent Z-fighting
+    static constexpr float ZFightingOffset = 0.004f;
```

Comment:
update: Can this please be moved to an `AZ_CVAR` in the .cpp so we can tweak this potentially (there are other examples in the White Box Gem and the repo in general, I think this will be super useful to tweak if we need to)

Reply:
I'd also recommend updating the name to `PolygonViewOverlapOffset` (as this isn't really related to the z/depth value)